### DISCUSSION TITLE
Combining full-text and Title/abstract phases: issues

### DISCUSSION THREAD

**Discussion by j0sien** [asreview/asreview#1296] [Q&A]
when combining the t/a and full-text phase.

Sometimes I mark a records as irrelevant, because no full-text can be found. However, the content of the abstract may be relevant. doesn't that confuse the model if you mark a record as 'irrelevant' just because the full text is not available?

Sometimes I encounter an evidence summary from another paper. The abstract looks totally relevant, but after checking the full-text, the publication type is not what you are looking. If I then reject this abstract, doesn't the model think I reject the content? I could only see in the full-text phase that it was actually an evidence summary. Aren't these arguments for separating the Title-abstract and full-text phase after all?

@daob

**Comment by FelixWdm** [comment]
Hi @j0sien, yes, this can be confusing. You're correct that you want to train the model in a way it shows you more records that you would like see, so prevent excluding potentially relevant records. It might be the model recognises some pattern in the abstract we can't see. But I wouldn't count on it yet. So, I would label records relevant, if I want to have similar records show up in the remaining part of the screening. 

I hope this helps. Best Felix

**Reply by j0sien** [reply]
thanks for your prompt reply.

So this would mean I mark something as relevant, and then add a note telling to myself: 
exclude, wrong publication type or exclude, no full-text; making sure I export the notes.

So we keep the benefit of the algorithm seeing patterns in the abstracts we don't see (need to go to full-text instead).

**Reply by FelixWdm** [reply]
Yes, and it will depend on the total number of included studies whether or not this will slow you down. The alternative is marking them irrelevant and hoping that the remaining relevants will show up anyway, since they will be more similar than 'true irrelevant' ones. Either way, a small downside, but in the end still a faster screening process I would say. But please keep us posted on your findings!
